From f17ab255685b559711afb84a59ece4ec965ffe5e Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Mon, 8 Mar 2021 17:39:00 -0600
Subject: [PATCH 10/41] Revert "firmware: stratix10-svc: add
 COMMAND_AUTHENTICATE_BITSTREAM flag"

commit ca77573500d30c9ff39b8589d8bb66eac7ececd4 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.114-lts

This reverts commit c8f78dea5e30706baf854fd9d7395b69d0e3f931.

Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 include/linux/firmware/intel/stratix10-svc-client.h | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/include/linux/firmware/intel/stratix10-svc-client.h b/include/linux/firmware/intel/stratix10-svc-client.h
index f68024f25aeb..9cdc3417b433 100644
--- a/include/linux/firmware/intel/stratix10-svc-client.h
+++ b/include/linux/firmware/intel/stratix10-svc-client.h
@@ -51,17 +51,12 @@
 #define SVC_STATUS_NO_SUPPORT		6
 #define SVC_STATUS_INVALID_PARAM	7
 /**
- * Flag bit for COMMAND_RECONFIG, in bit number
+ * Flag bit for COMMAND_RECONFIG
  *
  * COMMAND_RECONFIG_FLAG_PARTIAL:
- * Set for partial FPGA configuration.
- *
- * COMMAND_AUTHENTICATE_BITSTREAM:
- * Set for bitstream authentication, which makes sure a signed bitstream
- * has valid signatures before committing it to QSPI flash memory.
+ * Set to FPGA configuration type (full or partial).
  */
-#define COMMAND_RECONFIG_FLAG_PARTIAL	0
-#define COMMAND_AUTHENTICATE_BITSTREAM	1
+#define COMMAND_RECONFIG_FLAG_PARTIAL	1
 
 /**
  * Timeout settings for service clients:
-- 
2.18.1

